Uplift/Services/Networking/Network.swift

```diff
@@ -16,3 +18,13 @@ final class Network {
     static let client = ApolloClient(url: UpliftEnvironment.baseURL)
 
 }
+
+class NetworkState: ObservableObject {
+
+    func handleCompletion(_ completion: Subscribers.Completion<Error>) {
+        if case let .failure(error) = completion {
+            Logger.services.critical("Error")
+        }
+    }
+
+}
```

Comment:
Not a bad idea to have this `handleCompletion` function, but we can just have this if case statement inside of the .`sink` closure so that we can do more things if the call fails. Let's remove this.